docs/basics/networkvariable.md

```diff
@@ -43,11 +42,34 @@ Start               | OnNetworkSpawn
 Also, you should only set the value of a `NetworkVariable` when first initializing it or if it is spawned.  It is not recommended setting a `NetworkVariable` when the associated `NetworkObject` is not spawned.
 :::
 
-
 :::tip
 If you need to initialize other components or objects based on a `NetworkVariable`'s initial synchronized state, then you might contemplate having a common method that is invoked on the client side within the `NetworkVariable.OnValueChanged` callback (if assigned) and `NetworkBehaviour.OnNetworkSpawn` method.
 :::
 
+### Supported Types
+
+:::note
+While `NetworkVariable` supports both managed and [unmanaged](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/constraints-on-type-parameters#unmanaged-constraint)types, managed types do come with an amount of additional overhead. Efforts are made to minimize GC allocations for managed `INetworkSerializable` types (a new value is only allocated if the value changes from `null` to non-`null`), but the ability of a type to be `null` adds additional overhead both in logic (checking for nulls before serializing) and bandwidth (every serialization carries an additional byte indicating whether or not the value is `null`).
+
+Also note that any type which contains a managed type is itself a managed type - so a struct that contains `int[]` is a managed type because `int[]` is a managed type.
+:::
+
+`NetworkVariable` provides support for the following types:
+
+* C# unmanaged [primitive types](../advanced-topics/serialization/cprimitives.md): `bool`, `byte`, `sbyte`, `char`, `decimal`, `double`, `float`, `int`, `uint`, `long`, `ulong`, `short`, and `ushort`, which will be serialized by direct memcpy into/out of the buffer.
+
+* Unity unmanaged [built-in types](../advanced-topics/serialization/unity-primitives.md): `Vector2`, `Vector3`, `Vector2Int`, `Vector3Int`, `Vector4`, `Quaternion`, `Color`, `Color32`, `Ray`, `Ray2D`, which will be serialized by direct memcpy into/out of the buffer.
+
+* Any [`enum`](../advanced-topics/serialization/enum-types.md) types, which will be serialized by direct memcpy into/out of the buffer.
+
+* Any type (managed or unmanaged) that implements [`INetworkSerializable`](../advanced-topics/serialization/inetworkserializable.md), which will be serialized by calling their `NetworkSerialize` method. **On the reading side, these values are deserialized in-place, meaning the existing instance will be reused and any non-serialized values will be left in their current state.**
```

Comment:
> You can serialized any type (managed or unmanaged) that implements [`INetworkSerializable`](../advanced-topics/serialization/inetworkserializable.md) by calling their `NetworkSerialize` method. **On the reading side, these values are deserialized in place, meaning the existing instance will be reused, and any non-serialized values will be left in their current state.**

Reply:
This isn't accurate - the NetworkSerialize method is called automatically by the SDK, it's not a user requirement for them to call it.